standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -1239,6 +1239,21 @@ public enum ConfVars {
       "will be dropped. The second piece in partition management is retention period. When 'discover.partition'\n" +
       "is set to true and if 'partition.retention.period' table property is defined, partitions that are older\n" +
       "than the specified retention period will be automatically dropped from metastore along with the data."),
+
+    // Statistics management task params
+    STATISTICS_MANAGEMENT_TASK_FREQUENCY("metastore.statistics.management.task.frequency",
+      "metastore.statistics.management.task.frequency",
```

Comment:
why the frequency set to 365 days? is is too long

Reply:
It's the requirement on the HIVE-28755 Jira.
From my perspective, the purpose of this PR is to reduce the size of all the column stats and to prevent the affect on performance. And the size of the column stats might take quite long time to stack up. So 365 days might be an appropriate time.